### PR TITLE
Make additions and edits to community rules

### PR DESCRIPTION
**Pull request by thatblindgeye**
#### Because:
Made several additions and edits to the community rules to add clarification as well as rules that would prove to be useful based on server interaction.

#### This commit

- **Rule 2:** Broke up the rule to two parts, mainly to add in verbiage to help prevent users from using "technically this is within scope" when asking for help on persona/ projects/homework. 

- **Rule 9, "Proper Channel" sub-rule**: Added verbiage about how the general channel on the server works, to help prevent users from trying to redirect others to different channels when that isn't appropriate.

- **Rule 9, screenshots sub-rule**: Added a sub-rule regarding posting screenshots instead of photos from a phone camera, and a couple of example of when screenshots might be more appropriate than code blocks. This makes it clear that photos are not adequate and users should look into how to take screenshots on their OS. 

- **Rule 10, constructive sub-rule:** Added a sub-rule to let users know that they should be constructive when offering feedback instead of trying to tear a user down for something like "your code is terrible" and nother else. This should prod users into thining about their feedback a little more and actually being helpful with it.

- **Rule 11:** Added another rule that just lets users know that while certain things may not be ban worthy, or may not really be a rule per se, that multiple instances of a user causing issues may justify a ban regardless and that they should always try changing their attitudes when called out on things by moderators. 

#### Checklist
 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/theodinproject/wiki/Contributing-Guide)?
 - [ ] You have verified the tests and linters all pass against your changes?
 - [ ] You have included automated tests for your changes where applicable?

### CODE REVIEW

File: app/views/static_pages/dashboard_steps/community_rules.md [hunk 1]
```diff
@@ -7,45 +7,58 @@ Our moderation team is committed to providing a friendly, safe and welcoming env
...
 
 #### 1. Please leave moderation to the moderators
+
 Please DM the @ModMail user with issues that need to be resolved with things on the server, like someone being a troll, or breaking rules in a way that requires intervention. Don't be afraid to let us know if something isn't right.
 
 #### 2. This community only supports the content in our curriculum
-All questions and advice should stay within the scope of our curriculum because we are only equipped to support our own material and recommendations. If you seek support on homework, personal projects, relationships, depression, mental illness, or anything else outside the scope of our curriculum, please seek out more equipped resources.
+
+All questions and advice should stay within the scope of our curriculum because we are only equipped to support our own material and recommendations. If you seek support on homework or personal projects, you should ask your teacher, a classmate, or another resource for help, even if the help you seek is for something covered by our curriculum.
```
Comment (rlmoser99) [inline app/views/static_pages/dashboard_steps/community_rules.md:15]: Good insight, but I think the wording is a bit hard to follow. What about something like this:
```suggestion
All questions and advice should stay within the scope of our curriculum because we are only equipped to support our own material and recommendations. Therefore, we do not support homework or personal projects, even if the topic is covered by our curriculum. You should ask your teacher, a classmate, or another resource for help.
```

File: app/views/static_pages/dashboard_steps/community_rules.md [hunk 2]
```diff
@@ -7,45 +7,58 @@ Our moderation team is committed to providing a friendly, safe and welcoming env
...
 
-* **Use external services for code sharing.** Posting small snippets of code is acceptable, but anything longer than a few lines is hard to read and parse. Do not upload files for people to download. Use [Pastebin](http://pastebin.com/) for long error messages. Use [CodePen](https://codepen.io/), [Replit](https://replit.com/), or [CodeSandbox](https://codesandbox.io/) for code examples.
+-   **Post screenshots when necessary, not photos.** If you need to share an image to provide additional context when seeking help, such as to show what your terminal or an error says, always post a screenshot using the default method for your computer or with a third party screenshot tool. Taking a picture of your screen from your phone provides a lower quality image, making it difficult for anyone to actually help.
```
Comment (rlmoser99) [inline app/views/static_pages/dashboard_steps/community_rules.md:61]: More great insight here. However, I think we should combine the two "do nots" together, because it feels buried inside the other guideline and end up making three bullet points. Also, I want to make sure you know that this information is expanded on in the proposed https://www.theodinproject.com/how_to_ask, under the "Provide the Relevant Source" section.
```suggestion
-   **Use external services to share code.** Posting small snippets of code is acceptable, but anything longer than a few lines is hard to read and parse. Use [CodePen](https://codepen.io/), [Replit](https://replit.com/), or [CodeSandbox](https://codesandbox.io/) to share code.

-   **Use screenshots to share other relevant information.** When you need to share an image of your terminal window or an error message, use a screenshot tool on your computer. If you are not familiar with this tool, you should research it using your favorite search engine. You can also use [Pastebin](http://pastebin.com/) to share long error messages.

-   **Do not share files or a picture of your computer screen.** Files can contain malware, so do not share files for people to download. A picture of a computer screen provides a low-quality image that is difficult for others to read.
```

File: app/views/static_pages/dashboard_steps/community_rules.md [hunk 3]
```diff
@@ -55,24 +68,30 @@ We expect everyone to handle themselves professionally and treat others respectf
...
+-   **Remember the Human.** Behind every username there is a person with feelings, so be kind to everyone! If you wouldn't say it out loud, or wouldn't want someone else saying it to you, don't post it.
+
+-   **Be gracious during misunderstandings.** When misunderstandings happen, choose to be gracious instead of argumentative. Give others the benefit of the doubt during text-based communication because you can not take into account body language, cultural differences, or other contextual cues.
 
 </div>
+
+#### 11. Be willing to change
+
+Not everything will result in a ban, but a lot of small things do add up. If you have been repeatedly told by a moderator to stop doing something, especially if your attitude and interactions within the community begin causing a toxic or unwelcoming environment, you may wind up being banned if you've proven to show zero effort in making changes. When you're informed about some form of behavior by a moderator, be open and willing to change your attitude to better fit the goals we wish to meet for our community.
```
Comment (rlmoser99) [inline app/views/static_pages/dashboard_steps/community_rules.md:97]: I do not think this needs to be a rule, because there is nothing to enforce. However, I like the jist of what you are saying and think it would fit better as a conclusion, because it is similar to the introduction above the rules.

```suggestion
Our moderation team enforces these rules to keep our community a friendly, safe and welcoming environment for everyone. If anyone repeatedly breaks rules, responds with negative/toxic attitudes, pushes the boundaries of our rules, or other “[help vampire](https://slash7.com/2006/12/22/vampires/)” behaviors, they will be banned.

If you are notified about your behavior by a moderator, we encourage you to be open and willing to change. Our rules are very similar to the expectations in professional workplaces, so this will better prepare you to interact professionally with your future teammates.
```

**Review by rlmoser99 — CHANGES_REQUESTED** [review]
I really like the insights that you've added, but nit picked some of the sentences. Feel free to take my suggestions and put your own spin on it. Just be careful to minimize any run-on sentences, because we have lots of people where english is not their first language.

File: app/views/static_pages/dashboard_steps/community_rules.md (hunk 2 above)
Comment (thatblindgeye) [reply app/views/static_pages/dashboard_steps/community_rules.md]: For this, the changes I made align with what you have here with some tweaks.

- I think including the Pastebin link with the "external sources" bullet makes more sense as they're all related in sharing text snippets.

- I updated the screenshot text to what you have, but kept the "don't share photos" verbiage there as well (as opposed to the files bullet below it). Again, I think it's more related to screenshots than file uploads.

Let me know what you think of the updated verbiage and bullets in this section.

Comment (rlmoser99) [reply app/views/static_pages/dashboard_steps/community_rules.md]: Cool. I think the pastebin link fits in either place and debated it back and forth myself.

**Review by rlmoser99 — APPROVED** [review]
These changes look good. Thanks for taking the time to consider my input and coming to a good middle ground. 👍
